src/Vary/Core.hs

```diff
@@ -15,10 +22,10 @@ import Data.Kind (Type)
 import Data.Typeable (Typeable, typeOf)
 import GHC.Exts (Any)
 import GHC.Generics
-import Unsafe.Coerce qualified as Data.Coerce
+import qualified Unsafe.Coerce as Data.Coerce
```

Comment:
accidental change?

Reply:
I hadn't added `ImportQualifiedPost` so this change was needed.